packages/tocco-theme/src/ToccoTheme/DefaultTheme.js

```diff
@@ -1,141 +1,93 @@
+    },
+    warning: {
+      paper: '#FFE082', // Amber 200
+      text: '#FFA000' // Amber 700
+    }
   }
 }
 
+/* FONT Family
+ * Use web safe fonts or provide font files and @font-face separately.
+ * Variable fontFamily control declaration but does not load fonts.
+ */
 const fontFamily = {
-  sansSerif: '"Helvetica Neue", Helvetica, Arial, sans-serif',
+  regular: '"Helvetica Neue", Helvetica, Arial, sans-serif',
   monospace: 'Menlo, Monaco, Consolas, "Courier New", monospace'
 }
+
 /* FONT SIZES
- * Unit 'rem' is required for correct presentation, independently from zoom and nesting.
- * It is recommended to use a rhythmic scale to ensure a harmonic typography.
+ * All font sizes are calculated automatically as an exponential scale.
+ * Both variables are unitless factors. Base font size is interpreted as
+ * rem, hence it inherits from websites root.
  */
-const fontSizes = [
-  '.7rem', '1rem', '1.4rem', '1.4rem', '2rem', '2.8rem', '3.9rem', '5.6rem'
-]
+const fontSizeBase = 1.4
+const fontSizeScale = 1.3
 
+/* FONT WEIGHTS
+ * Change weights for extra light or extra bold typography but ensure that
+ * font files does provide such weights.
+ */
 const fontWeights = {
-  light: 300,
   regular: 400,
   bold: 700
 }
 
 /* LINE HEIGHT
-   lineHeights[0] = dense mode
-   lineHeights[1] = normal mode
+ * Line height is a factor applied on font size.
  */
-const lineHeights = [
-  1,
-  1.4
-]
-
-const outline = 'none'
-
-const overlays = {
-  disabled: {
-    color: colors.base.paper,
-    opacity: 0.7
-  }
+const lineHeights = {
+  dense: 1,
+  regular: 1.4
 }
 
-const radii = [
-  0, '2px', '4px', '8px', '16px'
-]
-
-const shadows = {
-  levels: [
-    'none',
-    '0 1px 1px 1px',
-    '0 2px 2px 2px'
-  ],
-  color: 'rgba(33, 33, 33, .6)'
-}
+/* RADII
+ * Radii is used to round corners of elements like panels, field and buttons.
+ */
+const radii = '4px'
 
 /* SPACE
- * Space controls vertical and horizontal white space rhythm.
- * Unit 'rem' is required for correct presentation, independently from zoom and nesting.
- * It is recommended to use powers of two to ensure alignment.
- * space[0] = 0px
- * space[1] = 1px
- * space[2] = 2px
- * space[3] = 4px
- * space[4] = 8px
- * space[5] = 16px
- * space[6] = 32px
- * space[7] = 64px
- * space[8] = 128px
- * space[9] = 256px
- * space[10] = 512px
+ * SpaceBase and spaceScale are used to control vertical and horizontal white
+ * space rhythm. All spaces are calculated automatically as an exponential scale.
+ * Both variables must be provided as unitless factors. SpaceBase is interpreted as
+ * rem, hence it inherits from websites root. It is recommended to use powers
+ * of two to ensure alignment.
  */
-const space = [
-  0, '.0625rem', '.125rem', '.25rem', '.5rem', '1rem', '2rem', '4rem', '8rem', '16rem', '32rem'
-]
+const spaceBase = _round(fontSizeBase * lineHeights.regular, 3)
+const spaceScale = 2
 
 const defaultTheme = {
   colors,
-  fontSizes,
+  fontSizeBase,
```

Comment:
IMO this file would be easier to read if you directly export an object rather than building an object with consts and then export it